doc/eclipse.html

```diff
@@ -104,10 +110,12 @@ <h2 align="center">TestNG Eclipse plug-in</h2>
 <li><b>Verbose</b>: enable the runtime TestNG verbose log</li>
 <li><b>Debug</b>: enable more runtime TestNG debug info</li>
 <li><b>Serialization Protocol</b>: the serialization protocol used for communicating between TestNG Eclipse Plugin and TestNG runtime.
-The default is "Object Serialization", the other option ("String Serialization") is deprecated.</li>
-<li><b>Add the POM's JVM Arguments when launch</b>: for maven based project, we normally define maven-surefire-plugin for test phase and maven-failsafe-plugin for integration test phase, there might be having the testing runtime JVM arguments defined in the "argLine" element of the maven plugin configuration. If you check this option, TestNG Eclipse Plugin will try to recognize the JVM arguments in "argLine", add them to TestNG runtime JVM arguments when launch.<br/>
-NOTE 1: these JVM arguments are put on front of user defined VM arguments on "Arguments" tab, which means you can overide the POM defined JVM argumetns with the ones on "Arguments" tab.<br/>
-NOTE 2: if there are multiple "argLine" elements in the pom.xml, the first presence will be used. If it's not the desired behaviour, you can simply uncheck this option, and type your JVM arguments on the "Arguments" tab next to this "Test" tab.</li>
+  <ul>
+    <li><b>Json Serialization</b>: the <i>default</i> protocol for packaging the message data in JSON format. it's introduced since TestNg Eclipse Plugin 6.9.11 to better fit with communicating to variant JRE versions running TestNG.</li>
+    <li><b>Object Serialization</b>: the protocol for packing the message data with Java Object I/O Stream.</li>
+    <li>String Serialization: is <i>deprecated</i>.</li>
+  </ul>
```

Comment:
- This protocol was introduced in the TestNG Eclipse plug-in 6.9.11 to better communicate with a different JRE running TestNG.
- This protocol packs the message data with Java serialization.
- Deprecated.